b2share/modules/records/cli.py

```diff
@@ -45,12 +44,16 @@
     as update_expired_embargoes_task
 from .utils import list_db_published_records
 from b2share.modules.handle.proxies import current_handle
+from flask import current_app
+
+from ...utils import get_base_url
```

Comment:
It would be better to use exact path here and not `...utils` to make it clearer, but this does not affect review.

Reply:
The exact location is now changed to `b2share.utils` with the latest commit.

Reply:
This should now also be changed to `b2share.utils`.